tracee-ebpf/tracee/argprinters.go

```diff
@@ -43,3 +43,37 @@ func PrintAlert(alert alert) string {
 
 	return res
 }
+
+func ParseMknodMode(mode uint16) string {
```

Comment:
It's more appropriate to put this functino in libbpfgo helpers (https://github.com/aquasecurity/libbpfgo/blob/main/helpers/argumentParsers.go) instead of here

Reply:
opened a PR in libbpgo

Reply:
Why not using the ParseInodeMode() already defined in the helpers pacakge of libbpfgo?

Reply:
ParseInodeMode is not suitable because in this case the mode is different format.

the way to parse the mode in this case is like the kernel does it here:
https://elixir.bootlin.com/linux/latest/source/fs/namei.c#L3711

Reply:
How is this a different format? This is mode_t, and the values are the same